src/main/java/com/artipie/asto/google/GoogleStorage.java

```diff
@@ -46,14 +44,25 @@
  *
  * @since 0.17
  * @checkstyle ConstantUsageCheck (500 lines)
+ * @checkstyle ParameterNumberCheck (500 lines)
  */
 @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
 public final class GoogleStorage implements Storage {
 
     /**
-     * API GET_URL.
+     * API GET_REQUEST.
      */
-    private static final String GET_URL = "https://storage.googleapis.com/storage/v1/b/%s/o/%s";
+    private static final String GET_REQUEST = "/storage/v1/b/%s/o/%s";
```

Comment:
@Guseyn why do we need this constant? As I understand this string is used only once in `value()` method to format request URL. I'd move to the method, it will be easier to understand what request URL we're using in HTTP client.

Reply:
@g4s8 I think it will be used in other methods, and I will rename it later on if needed. I think it would be easier to keep as it is.